modules/library/coverage/src/main/java/org/geotools/coverage/GridSampleDimension.java

```diff
@@ -135,21 +117,9 @@
      */
     private final boolean hasQuantitative;
 
-    /**
-     * The {@link Category#getSampleToGeophysics sampleToGeophysics} transform used by every
-     * quantitative {@link Category}, or {@code null}. This field may be null for two reasons:
-     *
-     * <ul>
-     *   <li>There is no quantitative category in this sample dimension.</li>
-     *   <li>There is more than one quantitative category, and all of them don't use the same
-     *       {@link Category#getSampleToGeophysics sampleToGeophysics} transform.</li>
-     * </ul>
-     *
-     * This field is used by {@link #getOffset} and {@link #getScale}. The
-     * {@link #getSampleToGeophysics} method may also returns directly this
-     * value in some conditions.
-     */
-    private final MathTransform1D sampleToGeophysics;
+    private double scale = 1;
```

Comment:
They are there to implement the sampleDimension interface which has getScale and getOffset. They are now simple metadata which can be attached to a sampleDimension.